MBBSEmu/Program.cs

```diff
@@ -199,6 +206,9 @@ static void Main(string[] args)
                 var resourceManager = _serviceResolver.GetService<IResourceManager>();
                 var globalCache = _serviceResolver.GetService<IGlobalCache>();
                 var fileHandler = _serviceResolver.GetService<IFileUtility>();
+                var channelDictionary = _channelDictionary;
```

Comment:
We should do a `GetService<<PointerDictionary<SessionBase>>()` here to keep the dictionary used throughout consistent.